src/libraries/System.Text.Json/tests/JsonElementReadTests.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+using System.Buffers;
+using System.IO;
+using System.Text.Encodings.Web;
+using System.Collections.Generic;
+
+namespace System.Text.Json.Tests
+{
+    public static class JsonElementReadTests
```

Comment:
nit: `JsonElementParseTests`?

Reply:
Sure. It was called "Read" because there are existing "Write" tests, but "Parse" is a bit more to the point.